graphql/resolvers.go

```diff
@@ -357,6 +359,49 @@ func (r *queryResolver) TaskFiles(ctx context.Context, taskID string) ([]*Groupe
 	return groupedFilesList, nil
 }
 
+func (r *mutationResolver) SetPriority(ctx context.Context, taskID string, priority int) (*restModel.APITask, error) {
+	t, err := task.FindOneId(taskID)
+	if err != nil {
+		return nil, ResourceNotFound.Send(ctx, err.Error())
+	}
+	if t == nil {
+		return nil, errors.Errorf("unable to find task %s", taskID)
+	}
+	authUser := gimlet.GetUser(ctx)
+	if priority > evergreen.MaxTaskPriority {
+		requiredPermission := gimlet.PermissionOpts{
+			Resource:      t.Project,
+			ResourceType:  "project",
+			Permission:    evergreen.PermissionTasks,
+			RequiredLevel: evergreen.TasksAdmin.Value,
+		}
+		taskAdmin := authUser.HasPermission(requiredPermission)
+		if !taskAdmin {
+			return nil, Forbidden.Send(ctx, fmt.Sprintf("Insufficient access to set priority %v, can only set priority less than or equal to %v", priority, evergreen.MaxTaskPriority))
+		}
+	}
+	if err = t.SetPriority(int64(priority), authUser.Username()); err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error setting task priority %v: %v", taskID, err))
```

Comment:
the last arg to `fmt.Sprintf` is `err` which will send the entire err object. doing `err.Error()` will send just the error message.